packages/blocks/src/image-block/image-model.ts

```diff
@@ -2,6 +2,7 @@ import { BaseBlockModel, defineBlockSchema } from '@blocksuite/store';
 
 import { BLOCK_BATCH } from '../surface-block/batch.js';
 import type { EdgelessBlockType } from '../surface-block/edgeless-types.js';
+import { groupRootId } from '../surface-block/elements/group/contants.js';
```

Comment:
I would argue that the block implementation should not depend on group related modules.

Reply:
okay